Makefile

```diff
@@ -87,8 +87,11 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	sed -i "s|STATUS_REPORTER_IMAGE_VALUE|$(IMG)|g" config/manager/manager.yaml
 	cd config/default && $(KUSTOMIZE) edit set image rbac-proxy=$(RBAC_PROXY_IMG)
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
+	# Revert back the STATUS_REPORTER_IMAGE_VALUE var
+	sed -i "s|value: $(IMG)|value: STATUS_REPORTER_IMAGE_VALUE|g" config/manager/manager.yaml
```

Comment:
Can't you just pipe the output of  `kustomize build` to `awk` and then to `kubectl`?